tests/sdk/sdk_integration_test.py

```
@@ -0,0 +1,106 @@
+def honcho_test_client(client: TestClient) -> Generator[Honcho, None, None]:
+    """
+    Returns a Honcho SDK client configured to talk to the test API.
+    """
+    http_client = httpx.Client(
+        transport=client._transport,  # pyright: ignore
+        base_url=str(client.base_url),
+        headers=client.headers,
+    )
+
+    honcho_client = Honcho(workspace_id="sdk-test-workspace", http_client=http_client)
+    yield honcho_client
```

Ruff (0.11.9)

No teardown in fixture `honcho_test_client`, use `return` instead of `yield`

Replace `yield` with `return`

(PT022)